components/collection/drop/Generative.vue

```diff
@@ -230,21 +180,9 @@ const mintedCount = computed(() =>
   Math.min(mintedDropCount.value, maxCount.value),
 )
 
-const mintedPercent = computed(() => {
-  const percent = (mintedCount.value / maxCount.value) * 100
-  return Math.round(percent)
-})
-
-const mintCountAvailable = computed(() => mintedCount.value < maxCount.value)
-
 const mintButtonDisabled = computed(
   () =>
-    isLogIn.value &&
-    Boolean(
-      !mintCountAvailable.value ||
-        !selectedImage.value ||
-        disabledByBackend.value,
-    ),
+    isLogIn.value && Boolean(!selectedImage.value || disabledByBackend.value),
 )
```

Comment:
The condition was changed here. I would like to manage this status in one place instead of two files.

Reply:
@Jarsen136 updated